boards/arm/96b_ivy5661/board.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018, UNISOC Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+/* Push button switch 2 */
+#define SW2_GPIO_PIN	6    /* GPIO22/Pin15 */
```

Comment:
We should move this into the device tree, take a look at this commit to see an example:

commit 619049a74c8b7179ca4925f243597359e4930640
Author: Kumar Gala <kumar.gala@linaro.org>
Date:   Fri Sep 28 16:35:16 2018 -0500

    boards: nrf51_ble400: Convert to using dts for led/button

Reply:
This modification will take some times.